x/gov/keeper/tally_test.go

```diff
@@ -349,7 +361,17 @@ func TestTallyDelgatorMultipleOverride(t *testing.T) {
 
 	require.False(t, passes)
 	require.False(t, burnDeposits)
-	require.False(t, tallyResults.Equals(types.EmptyTallyResult()))
+	valSelfDelegations := []sdk.Int{
+		app.StakingKeeper.TokensFromConsensusPower(ctx, valPowers[0]),
+		app.StakingKeeper.TokensFromConsensusPower(ctx, valPowers[1]),
+		app.StakingKeeper.TokensFromConsensusPower(ctx, valPowers[2]),
+	}
+	require.Equal(t, tallyResults.String(), types.NewTallyResult(
+		valSelfDelegations[0].Add(valSelfDelegations[1]).Add(valSelfDelegations[2]),
+		sdk.ZeroInt(),
+		delTokens.Add(delTokens),
+		sdk.ZeroInt(),
+	).String())
 }
 
 // As validators can only vote with their own stake, delegators don't inherit votes from validators
```

Comment:
would you be able to improve the comment above accordingly? also for the other test. Otherwise TY! More robust tests are always better!

Reply:
The comment is still accurate as far as I understand!